src/components/mgt-people/mgt-people.ts

```diff
@@ -62,6 +84,35 @@ export class MgtPeople extends MgtTemplatedComponent {
   })
   public groupId: string;
 
+  /**
+   * user id array
+   *
+   * @memberof MgtPeople
+   */
+  @property({
+    attribute: 'user-ids',
+    converter: (value, type) => {
+      return value.split(',');
+    }
+  })
+  public userIds: string[];
+
+  /**
+   * Sets how the person-card is invoked
+   * Set to PersonCardInteraction.none to not show the card
+   *
+   * @type {PersonCardInteraction}
+   * @memberof MgtPerson
+   */
+  @property({
+    attribute: 'person-card',
+    converter: (value, type) => {
+      value = value.toLowerCase();
+      return PersonCardInteraction[value] || PersonCardInteraction.none;
+    }
+  })
+  public personCardInteraction: PersonCardInteraction = PersonCardInteraction.none;
```

Comment:
Default should be hover as that's what it was previously. Otherwise it would be a breaking change

Reply:
fixed as part of: d0a7d85